packages/cln_plugin/lib/src/rpcmethods/manifest/types/notification.dart

```diff
@@ -0,0 +1,11 @@
+class Notification {
+  late String _method;
+
+  Notification(String method) {
+    _method = method;
+  }
+
+  String getMethod() {
+    return _method;
+  }
+}
```

Comment:
Too simple to be a class, the notification can be mapped inside the GetManifest as `HasMap<String, Function>`

where the key is the `method` and the value is the callback!